### PR TITLE
MathGame

### PR DESCRIPTION
**Pull request by InfiniteBlanK3T**

### CODE REVIEW

**Comment by TheCSharpAcademy** [comment]
@InfiniteBlanK3T Hi there! I can't see this project submitted in The C# Academy platform. Could you please do that so I can review it? 😊

**Comment by InfiniteBlanK3T** [comment]
Submitted!

File: InfiniteBlanK3T.MathGame/Program.cs [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+        {
+            Console.WriteLine($"Wrong!. Right answer: {result}");
+            return false;
+        }
+        Console.WriteLine("Correct!");
+        return true;
+    }
+
+    static bool Division(List<int> aList)
+    {
```
Comment (TheCSharpAcademy) [inline InfiniteBlanK3T.MathGame/Program.cs:257]: This division shouldn't be presented to the user, since it results in a non-integer: 
image

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

**Comment by TheCSharpAcademy** [comment]
@InfiniteBlanK3T Let me know if you need any help?

**Comment by TheCSharpAcademy** [comment]
@InfiniteBlanK3T We haven't heard from you in a while 😔

**Comment by TheCSharpAcademy** [comment]
@InfiniteBlanK3T I'm closing the PR as it's been reviewed more than a month ago, feel free to resubmit the project and reopen the PR if you'd like to proceed.

**Pull request closed without merging**